Collecting anaconda-cloud-auth
  Downloading anaconda_cloud_auth-0.7.2-py3-none-any.whl.metadata (13 kB)
Requirement already satisfied: anaconda-cli-base>=0.4.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (0.5.2)
Requirement already satisfied: cryptography>=3.4.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (43.0.3)
Requirement already satisfied: keyring in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (24.3.1)
Requirement already satisfied: pkce in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (1.0.3)
Requirement already satisfied: pydantic in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (2.10.3)
Requirement already satisfied: pyjwt in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (2.10.1)
Requirement already satisfied: python-dotenv in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (0.21.0)
Requirement already satisfied: requests in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (2.32.3)
Requirement already satisfied: semver<4 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cloud-auth) (3.0.2)
Requirement already satisfied: click in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (8.1.7)
Requirement already satisfied: pydantic-settings>=2.3 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (2.6.1)
Requirement already satisfied: readchar in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (4.0.5)
Requirement already satisfied: rich in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (13.9.4)
Requirement already satisfied: tomli in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (2.0.1)
Requirement already satisfied: typer in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (0.9.0)
Requirement already satisfied: cffi>=1.12 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from cryptography>=3.4.0->anaconda-cloud-auth) (1.17.1)
Requirement already satisfied: jaraco.classes in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from keyring->anaconda-cloud-auth) (3.2.1)
Requirement already satisfied: importlib-metadata>=4.11.4 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from keyring->anaconda-cloud-auth) (8.5.0)
Requirement already satisfied: annotated-types>=0.6.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from pydantic->anaconda-cloud-auth) (0.6.0)
Requirement already satisfied: pydantic-core==2.27.1 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from pydantic->anaconda-cloud-auth) (2.27.1)
Requirement already satisfied: typing-extensions>=4.12.2 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from pydantic->anaconda-cloud-auth) (4.12.2)
Requirement already satisfied: charset-normalizer<4,>=2 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from requests->anaconda-cloud-auth) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from requests->anaconda-cloud-auth) (3.7)
Requirement already satisfied: urllib3<3,>=1.21.1 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from requests->anaconda-cloud-auth) (2.3.0)
Requirement already satisfied: certifi>=2017.4.17 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from requests->anaconda-cloud-auth) (2025.1.31)
Requirement already satisfied: pycparser in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from cffi>=1.12->cryptography>=3.4.0->anaconda-cloud-auth) (2.21)
Requirement already satisfied: zipp>=3.20 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from importlib-metadata>=4.11.4->keyring->anaconda-cloud-auth) (3.21.0)
Requirement already satisfied: more-itertools in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from jaraco.classes->keyring->anaconda-cloud-auth) (10.3.0)
Requirement already satisfied: setuptools>=41.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from readchar->anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (75.8.0)
Requirement already satisfied: markdown-it-py>=2.2.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from rich->anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (2.2.0)
Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from rich->anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (2.15.1)
Requirement already satisfied: mdurl~=0.1 in /Users/FIOMAC/anaconda3/lib/python3.11/site-packages (from markdown-it-py>=2.2.0->rich->anaconda-cli-base>=0.4.0->anaconda-cloud-auth) (0.1.0)
Downloading anaconda_cloud_auth-0.7.2-py3-none-any.whl (24 kB)
Installing collected packages: anaconda-cloud-auth
Successfully installed anaconda-cloud-auth-0.7.2
